libs/markets/src/lib/components/markets-container/markets-container.tsx

```diff
@@ -30,11 +40,14 @@ export const MarketsContainer = ({ onSelect }: MarketsContainerProps) => {
     };
   }, [reload]);
 
+  const handleOnGridReady = useCallback(() => {
```

Comment:
suggestion (blocking): just tried out using useMemo to create col defs. This fixes the issue and is more simple

Reply:
you can merge this change

Reply:
merged and little bit improved